src/lib/fragments/AgGrid.react.js

```diff
@@ -648,93 +732,16 @@ export default class DashAgGrid extends Component {
     onGridReady(params) {
         // Applying Infinite Row Model
         // see: https://www.ag-grid.com/javascript-grid/infinite-scrolling/
-        const {
-            rowModelType,
-            selectedRows,
-            resetColumnState,
-            csvExportParams,
-            exportDataAsCsv,
-            selectAll,
-            deselectAll,
-            deleteSelectedRows,
-            filterModel,
-            setProps,
-            columnState,
-            paginationGoTo,
-        } = this.props;
+        const {rowModelType} = this.props;
 
-        const propsToSet = {};
         if (rowModelType === 'infinite') {
             params.api.setDatasource(this.getDatasource());
         }
 
-        this.setState({
+        this.setState(() => ({
```

Comment:
Is it necessary to make this a function?

Reply:
No, was trying to solve the bad `setState` issue, we can revert.